docs/modules/ROOT/pages/gitops/howto-argocd.adoc

```diff
@@ -0,0 +1,104 @@
+= Setting up an ArgoCD instance
+
+This how-to explains how to create a new ArgoCD instance on your cluster, leveraging the ArgoCD operator which comes preinstalled on APPUiO Managed OpenShift clusters.
+
+== Requirements
+
+You will need admin access to the cluster on which you want to deploy ArgoCD.
+
+Furthermore, this how-to assumes you have the following tools installed:
+
+`oc`:: You can download the OpenShift command directly from OpenShift Web Console. Open the help menu (marked as a question mark at the top right) and select the "Command line tools" entry.
+
+
+== Cluster Scope vs. Namespace Scope
+By default, an ArgoCD instance will be able to manage all resources within the same namespace it's deployed in, making it namespace scoped.
+This allows for easy setup of small-scope GitOps projects.
+
+However, ArgoCD is also capable of managing resources across multiple namespaces and cluster-scoped resources.
+To accomplish this, an ArgoCD instance needs to be registered as cluster scoped in the VSHN-managed ArgoCD operator.
+
+=== Requesting a Cluster Scope ArgoCD instance
+If you need to manage resources across multiple namespaces with the same ArgoCD instance, or you need to manage cluster-scope resources, you will need to request a cluster scope instance.
+
+Create a Service Desk Ticket with the following information:
+* State that you need a cluster-scope ArgoCD instance
+* Include the *namespace name* in which your ArgoCD instance will be deployed
+
+VSHN will register the provided namespace to allow for a cluster scope ArgoCD instance.
```

Comment:
here you use "cluster scope" while elsewhere you use cluster-scope -- I'm not sure which one is better/correct, but I'd prefer having one consistent spelling.

Reply:
time to actually look up hyphenation rules!

verdict: since the two words act as an adjective and convey a single idea, a hyphen should be used.